docs/docs_server_configuration_reference.md

```diff
@@ -198,3 +198,15 @@ The name of a localizaiton file in `data/localization` that would be used by `M.
   // ...
 }
 ```
+
+## minimumPlayersToStart
+
+The minimal amount of players to begin deathmatch
+
+```json5
+{
+  // ...
+  "minimumPlayersToStart": 5
```

Comment:
we need prefix like sweet or sweetPie to differ server settings and gamemode settings
We also need to tell users that settings is sweetpie only and is not about vanilla server